fulp_modules/features/antagonists/bloodsuckers/code/bloodsucker/bloodsucker_crypt.dm

```diff
@@ -740,7 +740,9 @@
 	var/rendered = span_cultlarge("<b>[user.real_name]:</b> [message]")
 	user.log_talk(message, LOG_SAY, tag=ROLE_BLOODSUCKER)
 	var/datum/antagonist/bloodsucker/bloodsuckerdatum = user.mind.has_antag_datum(/datum/antagonist/bloodsucker)
-	to_chat(bloodsuckerdatum.vassals, rendered)
+	for(var/datum/antagonist/vassal/receiver in bloodsuckerdatum.vassals)
+		var/mob/receiver_mob = receiver.owner?.current
+		to_chat(receiver_mob, rendered)
```

Comment:
```suggestion
	for(var/datum/antagonist/vassal/receiver as anything in bloodsuckerdatum.vassals)
		to_chat(receiver.owner.current, rendered)
```
we already know they have a mind and a body because otherwise they wouldnt be in the vassals lise
We also know that only vassal antag datums will be in the bloodsucker's vassals list, so we can as anything it.